first_order_model/reconstruction.py

```diff
@@ -228,6 +229,13 @@ def reconstruction(config, generator, kp_detector, checkpoint, log_dir, dataset,
         else:
             raise AttributeError('Checkpoint should be specified for reconstruction')
 
+    reload_gen = train_params.get('shrunk_gen', None)
```

Comment:
What exactly is this doing? can you clarify in a comment? What part of generator is it reloading? And can you use a better training parameter to clarify exactly what this means?